photon.py

```diff
@@ -171,17 +194,20 @@
 
 def intel_extractor(url, response):
     """Extract intel from the response body."""
-    matches = re.findall(r'([\w\.-]+s[\w\.-]+\.amazonaws\.com)|([\w\.-]+@[\w\.-]+\.[\.\w]+)', response)
-    if matches:
-        for match in matches:
-            verb('Intel', match)
-            intel.add(url + ':' + ''.join(list(match)))
+    for rintel in rintels:
+        res = re.sub(r'<(script).*?</\1>(?s)', '', response)

```

Comment:
I think that looking for intels inside html and javascript code should be done inside specific functions. Otherwise, some regex will spit out garbage by matching javascript nonsense.